bridges/ethereum/src/actors/block_relay_and_poi.rs

```diff
@@ -212,13 +215,22 @@ pub fn block_relay_and_poi(
                     let config = Arc::clone(&config);
                     let eth_state = Arc::clone(&eth_state);
                     move |(including, resolving, needs_relaying)| {
-
                         if (is_new_block && config.enable_block_relay_new_blocks) || (!is_new_block && config.enable_block_relay_old_blocks) {
                             // Optimization: do not process blocks that do not contain requests coming from ethereum
                             if including.is_empty() && resolving.is_empty() && !needs_relaying {
                                 log::debug!("Skipping empty superblock");
                                 return futures::finished(());
                             }
+                            let last_id =  if let Some(&(id, _, _, _, _)) = including.last() {
```

Comment:
There is a special syntax to "ignore the other fields"

```suggestion
                            let last_id =  if let Some(&(id, ..)) = including.last() {
```